projects/c-blosc/build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -eu
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# build project
+mkdir -p build_dir
+pushd build_dir
+cmake ..
+make -j$(nproc)
+make install
+popd
+
+find . -name "*.a"
+
+# build fuzzers
+for fuzzers in $(find $SRC -name '*_fuzzer.cc'); do
```

Comment:
Not a huge fan of looping when there's only one fuzzer

Reply:
Used looping because just in case the owner of C-blosc wants to add more fuzzers later on. I can remove it if you prefer without it.

Reply:
Meh. this is an OK reason for leaving it I guess. I generally don't do things like this unless I know for sure they will be used because of [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it)